src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -3176,6 +3176,19 @@ operand immL_32bits()
   interface(CONST_INTER);
 %}
 
+// Int Immediate: power of 2, positive
+operand immI_bitmask()
```

Comment:
`immI_bitmask` doesn't say much. What about `immI_Pow2M1`?

Reply:
👍 - Agreed, this is more descriptive.